sdks/python/apache_beam/typehints/schemas_test.py

```diff
@@ -91,12 +93,34 @@ def test_typing_survives_proto_roundtrip(self):
 
     test_cases = all_primitives + \
                  basic_array_types + \
-                 basic_map_types + \
-                 selected_schemas
+                 basic_map_types
 
     for test_case in test_cases:
       self.assertEqual(
-          test_case, typing_from_runner_api(typing_to_runner_api(test_case)))
+          test_case, typing_from_runner_api(
+              typing_to_runner_api(test_case,
+                                   schema_registry=SchemaTypeRegistry()),
+              schema_registry=SchemaTypeRegistry()
+          ))
+
+    for test_case in selected_schemas:
```

Comment:
We only break them out here because of the issue checking equality between NamedTuple types, in that test we can still just check for equality between the generated protos

Reply:
Makes sense, thanks!